package.json

```diff
@@ -3,10 +3,14 @@
   "version": "1.0.1",
   "description": "A thin layer over react-motion for animating routes in react-router.",
   "main": "lib/react-router-transition.js",
-  "module": "src/index.js",
-  "jsnext:main": "src/index.js",
+  "module": "es/index.js",
+  "jsnext:main": "es/index.js",
```

Comment:
I did some diff comparisons, looks good except we may want to consider changing the folder it builds to? es/ fine?

Reply:
yea, sgtm